src/main/java/picard/analysis/CollectInsertSizeMetrics.java

```diff
@@ -162,26 +164,27 @@ public static void main(final String[] argv) {
         }
         else  {
             file.write(OUTPUT);
-
-            final int rResult;
-            if(HISTOGRAM_WIDTH == null) {
-                rResult = RExecutor.executeFromClasspath(
-                    Histogram_R_SCRIPT,
-                    OUTPUT.getAbsolutePath(),
-                    Histogram_FILE.getAbsolutePath(),
-                    INPUT.getName());
-            } else {
-                rResult = RExecutor.executeFromClasspath(
-                    Histogram_R_SCRIPT,
-                    OUTPUT.getAbsolutePath(),
-                    Histogram_FILE.getAbsolutePath(),
-                    INPUT.getName(),
-                    String.valueOf(HISTOGRAM_WIDTH) ); //Histogram_WIDTH is passed because R automatically sets Histogram width to the last
-                                                         //bin that has data, which may be less than Histogram_WIDTH and confuse the user.
-            }
-
-            if (rResult != 0) {
-                throw new PicardException("R script " + Histogram_R_SCRIPT + " failed with return code " + rResult);
+            if(Histogram_FILE != null) {
+                final int rResult;
+                if (HISTOGRAM_WIDTH == null) {
+                    rResult = RExecutor.executeFromClasspath(
+                            Histogram_R_SCRIPT,
+                            OUTPUT.getAbsolutePath(),
+                            Histogram_FILE.getAbsolutePath(),
+                            INPUT.getName());
+                } else {
+                    rResult = RExecutor.executeFromClasspath(
+                            Histogram_R_SCRIPT,
+                            OUTPUT.getAbsolutePath(),
+                            Histogram_FILE.getAbsolutePath(),
+                            INPUT.getName(),
+                            String.valueOf(HISTOGRAM_WIDTH)); //Histogram_WIDTH is passed because R automatically sets Histogram width to the last
```

Comment:
Please move the two `//` comments so they precede the line they comment on, and add a space after the `//`, e.g.
```
    INPUT.getName(),
    // Histogram_WIDTH is passed because R automatically sets Histogram width to the last
    // bin that has data, which may be less than Histogram_WIDTH and confuse the user.
    String.valueOf(HISTOGRAM_WIDTH));
```

Reply:
Reformatted, thanks.